frontend/cypress/integration/authRedirects.spec.js

```diff
@@ -0,0 +1,28 @@
+import faker from 'faker'
+
+describe('Validates authentication redirects', () => {
+  const username = faker.name.firstName()
+  const email = faker.internet.email()
+  const password = '123456'
+
+  before(() => {
+    cy.visit('/')
+    cy.createUser(username, email, password)
+  })
+  it("Redirects unauthenticated users to '/login' when accessing reqSignIn page", () => {
+    // `/project/new` is marked as `reqSignIn`.
+    cy.visit('/projects/new')
+    cy.url().should('eq', 'http://kitspace.test:3000/login')
+  })
+
+  it('Redirects authenticated users to homepage when accessing reqSignOut page', () => {
```

Comment:
I think we should re-direct to /login when signing out. The main reason someone signs out manually is because they want to sign in with a different account.

Reply:
Easy fix!